schema.sql

```diff
@@ -7,5 +7,6 @@ vet_clinic=# CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL(10, 2)
+    weight_kg DECIMAL(10, 2),
+    species VARCHAR(255)
```

Comment:
I noticed you added the species column directly in your code😉. But this won't be feasible on your code editor😉. The table has already been created and we just want to add a new column. you did the right thing as shown in the PR description by using the `alter table` method😉. Kindly use that same method here to add the species column to your animal table😉.

you should use this instead after the create table `ALTER TABLE animals ADD COLUMN species VARCHAR(255);` I picked that from your PR description.

Reply:
Thank you dear @Ibizugbe Your review is very detailed.

Reply:
well resolved! ✔️